packages/cloud-cognitive/src/components/AboutModal/AboutModal.js

```diff
@@ -52,13 +47,12 @@ export let AboutModal = React.forwardRef(
       closeIconDescription,
       content,
       copyrightText,
-      legalText,
+      generalText,
```

Comment:
The new guidance for about modal uses the term `General text` to describe the content that goes here. Thoughts on this naming? I feel like this is equivalent to `description` that we use elsewhere. Alternatively, we could possibly have this just as `children`? 🤔

Reply:
Actually, it could serve the same purpose as `content` ... which if products are using example content currently, in the new design, it probably _should_ fall under the `General text` from guidance. So I think `content` might be more appropriate.